Topics/Tech_Stacks/Apache_Superset.md

```diff
@@ -38,6 +42,46 @@ This message shows up due to an update in version 2.1.0 to force secure configur
 1. The command `pip install apache-superset` doesn't work. This is because Apache Superset currently supports python version 3.8 and 3.9. Any python versions that's lower or higher will result in a failure.
 
 
+## Creating a Custom Plugin:
+
+**To get started on creating a custom plugin, you can follow the instruction [HERE](https://superset.apache.org/docs/contributing/creating-viz-plugins/)**
+
+There are example plugins [example](https://github.com/preset-io/superset-plugin-chart-liquid) which you can reference. Furthermore, this [youtube tutorial](https://www.youtube.com/watch?v=LDHFY9xTzls) can help you as well.
+
+### Potential Issues Creating a Custom Plugin:
+
+Note that one may get errors from `npm run build`, but those errors do not affect the actual building of the plugin. `npm` is a large package manager, and thus it yields irrelevant errors when trying to build the plugin. In case of version conflict errors with other tools under `npm`, it is recommended to use the `--force` flag, again due to the nature of `npm`.
+
+To add the package to Superset, go to the `superset-frontend` subdirectory in your Superset source folder (assuming both the `your-plugin` plugin and `superset` repos are in the same root directory) and run
```

Comment:
Can you elaborate on what is inside `superset` and `superset-frontend`, if any?

Reply:
I guess I was not clear. Once you install superset, you should have a folder called "superset". Then you go into the `superset/superset-frontend` directory. In that folder, run `npm i -S ../../your-plugin` once you ensure both the `your-plugin` plugin and `superset` repos are in the same root directory